src/charm.py

```diff
@@ -754,6 +760,23 @@ def _is_user_created(self, user: MongoDBUser) -> bool:
     def _set_user_created(self, user: MongoDBUser) -> None:
```

Comment:
is it possible to use "_is_user_created" from charm here? https://github.com/canonical/mongodb-operator/blob/0e75a6f4e8612116e7e22ef637f042826b563317/src/charm.py#L751

Reply:
No, and this code isn't relevant to this PR, plus `is_user_created`, relies `set_user_created`, we cannot replace it.

See [src/charm.py here](https://github.com/canonical/mongodb-operator/blob/0e75a6f4e8612116e7e22ef637f042826b563317/src/charm.py#L750C1-L755C75)